data/free worlds reconciliation.txt

```diff
@@ -940,7 +940,7 @@ mission "FW Syndicate Capture 1C"
 mission "FW Pug 1"
 	landing
 	name "What happened to Rand?"
-	description "Try to determine why communications with the Rand system were cut off. If you cannot reach Rand, at least look for anything odd in the neighboring systems."
+	description "Try to determine why communications with Rand, in the Zeta Aquilae system, were cut off. If you cannot reach Rand, at least look for anything odd in the neighboring systems."
```

Comment:
not sure if these commata are necessary

Reply:
This is structured the same way as the `<destination>` replacement, which reads `<planet> in the <system> system`, so the commas can be removed.

Reply:
My main concern was to name the system properly. If no-commas is more in line with the established style, no-commas it is.
```suggestion
	description "Try to determine why communications with Rand in the Zeta Aquilae system were cut off. If you cannot reach Rand, at least look for anything odd in the neighboring systems."
```